@types/foundry-pc-types/types/framework/entities/chatmessage.d.ts

```diff
@@ -0,0 +1,199 @@
+declare class Messages extends Collection<ChatMessage> {
+	entities: ChatMessage[];
+
+	/**
+	 * Elements of the Messages collection are instances of the ChatMessage class
+	 */
+	get object(): ChatMessage;
+
+	values(): IterableIterator<ChatMessage>;
+
+	/* -------------------------------------------- */
+	/*  Socket Listeners and Handlers
+	/* -------------------------------------------- */
+
+	/**
+	 * If requested, dispatch a Chat Bubble UI for the newly created message
+	 * @param response	The created ChatMessage response
+	 */
+	protected _sayBubble(response: object): void;
+
+	/**
+	 * Handle export of the chat log to a text file
+	 */
+	protected export(): void;
+
+	/**
+	 * Allow for bulk deletion of all chat messages, confirm first with a yes/no dialog.
+	 * @see {@link Dialog.confirm}
+	 */
+	flush(): Promise<any>;
+}
+
+/**
+ * The Chat Message class is a type of :class:`Entity` which represents individual messages in the chat log.
+ *
+ * @type {Entity}
+ */
+declare class ChatMessage extends Entity {
+	/**
+	 * Get a reference to the user who sent the chat message
+	 */
+	user: User;
+
+	/**
+	 * If the Message contains a dice roll, store it here
+	 */
+	protected _roll: any;
+
+	/**
+	 * Configure the attributes of the ChatMessage Entity
+	 */
+	static get config(): {
+		baseEntity: ChatMessage;
+		collection: Messages;
+		embeddedEntities: any;
+	};
+
+	/* -------------------------------------------- */
+	/*  Properties and Attributes
+	/* -------------------------------------------- */
+
+	/**
+	 * Return the recommended String alias for this message.
+	 * The alias could be a Token name in the case of in-character messages or dice rolls.
+	 * Alternatively it could be a User name in the case of OOC chat or whispers.
+	 */
+	get alias(): string;
+	/**
+	 * Return whether the ChatMessage is visible to the current user Messages may not be visible if they are private whispers
+	 */
+	get visible(): boolean;
```

Comment:
Codacy has a fix for the issue: [Expected blank line between class members.](https://app.codacy.com/gh/bmarian/token-tooltip-alt/pullRequest?prid=7194732)
```suggestion
	 */
```